harvester_e2e_tests/integration/test_vm_functions.py

```diff
@@ -101,14 +153,414 @@ def test_minimal_vm(api_client, image_id, unique_vm_name, wait_timeout):
+                                fails.append(
+                                    (vol_name, f"Failed to delete\nStatus({code}): {data}"))
+
+                        assert not fails, (
+                            f"Failed to delete VM({unique_vm_name})'s volumes with errors:\n"
+                            "\n".join(f"Volume({n}): {r}" for n, r in fails)
+                        )
+                        break
+                    sleep(3)
+                else:
+                    raise AssertionError(
+                        f"Failed to Start VM({unique_vm_name}) with errors:\n"
+                        f"Status({code}): {data}"
+                    )
+            else:
+                raise AssertionError(
+                    f"Failed to Update VM({unique_vm_name}) with errors:\n"
+                    f"Phase: {data.get('status', {}).get('phase')}\t"
+                    f"Status: {data.get('status')}\n"
+                    f"API Status({code}): {data}"
+                )
+            break
+        sleep(3)
+    else:
+        raise AssertionError(
+            f"Failed to create VM({cpu} core, {mem} RAM) with errors:\n"
+            f"Phase: {data.get('status', {}).get('phase','')}\t"
+            f"Status: {data.get('status', {})}\n"
+            f"API Status({code}): {data}"
+        )
+
+
+@ pytest.mark.p0
+@ pytest.mark.negative
+@ pytest.mark.virtualmachines
+def test_vm_with_bogus_vlan(api_client, image_id, unique_vm_name, wait_timeout, bogus_vlan_net):
+    """test building a VM with a VM (VLAN) Network has a bogus VLAN ID (no DHCP)
+
+    Prerequisite:
+        Setting opensuse-image-url set to a valid URL for
+        an opensuse image.
+
+    Manual Test Doc(s):
+        - N/A
+
+    Args:
+        api_client (HarvesterAPI): HarvesterAPI client_
+        image_id (str): corresponding image_id from fixture_
+        unique_vm_name (str): fixture at module level based unique vm name
+        wait_timeout (int): seconds for wait timeout from fixture
+        bogus_vlan_net (dict): the data dict that contains info surrounding vm net
+
+    Raises:
+        AssertionError: fails to create, delete, or delete volumes
+
+    Steps:
+    1. build vm with a single virtio network interface
+    that has a bogus vlan vm network (no dhcp)
+    2. delete vm and volumes
+
+    Expected Result:
+    - assert vlan vm network can be created successfully (fixture level)
+    - assert vm can be created successfully
+    - assert 'ipAddresses' not in the status of running vm's interfaces
+    - assert can delete vm and volumes
+    """
+    cpu, mem = 1, 2
+    bvn = bogus_vlan_net
+    net = {
+        "iface": dict(model='virtio', name='default'),
+        "network": dict(name='default')
+    }
+    net['iface']["bridge"] = dict()
+    net['network']['multus'] = dict(
+        networkName=f"{bvn['metadata']['namespace']}/{bvn['metadata']['name']}"
+    )
+    network_list = []
+    network_list.append(net)
```

Comment:
`net` and `network_list` is not used.

Reply:
I've cleaned those variables up.  Seem to have been hanging around from a previous iteration on the test function.  Good catch :smile: :+1: